### DISCUSSION TITLE
Upto which version of citus is distribution on foreign table supported?

### DISCUSSION THREAD

**Discussion by OkenKhuman** [citusdata/citus#7668] [Q&A] [answered]
I came across [old blog](https://www.citusdata.com/blog/2016/06/10/scaling-mysql-with-citus/).
I give it a shot on recent citus version but it seems the distribution on fdw based tables are now disabled. Please let me know upto which version this feature is supported.

**Comment by naisila** [comment, marked as answer]
Latest version supporting foreign distributed tables is Citus 10.2.
In order to simplify the user experience, we dropped support for foreign distributed or reference tables via [Pull request #5605](https://github.com/citusdata/citus/pull/5605). Instead, we enabled "citus managed local tables" to become foreign tables that can be accessed from any node via [Pull request #5461](https://github.com/citusdata/citus/pull/5461). This happened in [Citus 11.0](https://www.citusdata.com/updates/v11-0/#deprecated-features).
